ci/build_wheel.sh

```diff
@@ -56,7 +56,19 @@ fi
 
 cd "${package_dir}"
 
-python -m pip wheel . -w dist -vvv --no-deps --disable-pip-version-check
+
+PARALLEL_LEVEL=$(python -c \
+  "from math import ceil; from multiprocessing import cpu_count; print(ceil(cpu_count()/2))")
+
+SKBUILD_BUILD_OPTIONS="-j${PARALLEL_LEVEL}" \
+python -m pip wheel \
+    -w dist \
+    -vvv \
+    --no-deps \
+    --disable-pip-version-check \
+    --extra-index-url https://pypi.nvidia.com \
+    --global-option="-j${PARALLEL_LEVEL}" \
```

Comment:
This `--global-option` is breaking the installation process. It issues this warning:

```
WARNING: Implying --no-binary=:all: due to the presence of --build-option / --global-option.
```

which in turn disallows us from using binary wheels of packages like pylibraft-cu12 and rmm-cu12.

```suggestion
```